starter-code/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import '../styles/SearchBar.css'
+
+export default class SearchBar extends Component {
+
+  state = {
+    search: ""
+  }
+
+  filteredSearch = (evt) => {
+    let searchValue = evt.target.value
+    this.setState({search: `${searchValue}`})
```

Comment:
normally you should be able to do this with only one function declared in your parent component.